src/common/options/global.yaml.in

```diff
@@ -6875,3 +6875,18 @@ options:
   level: dev
   default: 0
   desc: When EC writes should generate PDWs (development only) 0=optimal 1=never 2=when possible
+- name: daemon_uniquifier
```

Comment:
Rather than config based, did we discard the idea of a new CLI param... or we don't want to alter the Ceph daemon CLI interface (backward compatibility)?